test_requirements.txt

```diff
@@ -12,4 +12,4 @@ ruamel.yaml<=0.16.6
 matplotlib>=3.0.0
 importlib_metadata
 tabulate
-ipython
\ No newline at end of file
+ipython
```

Comment:
we'd better not add the ipython module as a dependency

Reply:
Added

Reply:
It was introduced in previous PR, and it's need to pass the ut.